lib/bamboo/phoenix.ex

```diff
@@ -1,287 +1,289 @@
-defmodule Bamboo.Phoenix do
-  @moduledoc """
-  Render templates and layouts with Phoenix.
-
-  This module makes it very easy to render layouts and views using Phoenix.
-  Pass an atom (e.g. `:welcome_email`) as the template name to render HTML and
-  plain text emails. Use a string if you only want to render one type, e.g.
-  `"welcome_email.text"` or `"welcome_email.html"`.
-
-  ## Examples
-
-      defmodule Email do
-        use Bamboo.Phoenix, view: MyApp.EmailView
-
-        def text_and_html_email_with_layout do
-          new_email()
-          # You could set just a text layout or just an html layout
-          |> put_text_layout({MyApp.LayoutView, "email.text"})
-          |> put_html_layout({MyApp.LayoutView, "email.html"})
-          # Or you can set a layout for both html and text at the same time
-          |> put_layout({MyApp.LayoutView, :email})
-          # Pass an atom to render html AND plain text templates
-          |> render(:text_and_html_email)
-        end
-
-        def text_and_html_email_without_layouts do
-          new_email()
-          |> render(:text_and_html_email)
-        end
-
-        def email_with_assigns(user) do
-          new_email()
-          # @user will be available in the template
-          |> render(:email_with_assigns, user: user)
+if Code.ensure_loaded?(Phoenix) do
```

Comment:
Yes, I think at some point a `bamboo_phoenix` lib might be nice. For now this works great. Thank you so much!